generator/contributors.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+import * as yaml from 'js-yaml';
+import * as fs from './fsp.js';
```

Comment:
I think I mentioned this to sam at the time...

would it make more sense to use `fs-extra` here?
https://github.com/jprichardson/node-fs-extra

Reply:
I am mostly indifferent about that. I just was hesitant to include yet another dependency for something we can do.